src/transaction-flow/input/ProfileEditor/ProfileEditor.tsx

```diff
@@ -459,26 +467,33 @@ const ProfileEditor = ({ data, dispatch, onDismiss }: Props) => {
         dispatch({ name: 'setFlowStage', payload: 'transaction' })
         return
       }
+
       dispatch({
         name: 'startFlow',
-        key: `update-profile-${name}`,
+        key: `edit-profile-flow-${name}`,
```

Comment:
nit: flow is assumed so is doesn't really need to be added here

Reply:
I use 'flow' to differentiate between multistep process that is resumable. Otherwise if I open an input with he key 'edit-profile-{name}', it will overwrite the progress on the transaction flow.